### ISSUE TITLE
Panel: Screen reader announcing 'Required' to every option 'Panel - Small, right-aligned, fixed, with footer' pane.

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10803]
**User Experience:**
Screen reader users will get confused if screen reader announces every option as 'Required'.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Usages:**
Panel - Small, right-aligned, fixed, with footer.

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge and turn on narrator.
2. Navigate to 'Panel' control by using Tab key and activate it.
3. Dialogue pane will be appeared.
4. Navigate to 'Small Panel, Anchored Left, Fixed Width' variant by using Tab key and activate it.
5. Now navigate radio controls  by using Tab key and verify the issue.

**Actual Result:**
Screen reader announces 'Required' for every radio options. Screen reader announces "Option A radio button selected **required** one of four." and when focus moves to Option B screen reader announces "Option B radio button selected **required** two of four".

**Expected Result:**
Screen reader should not announces 'Required' for every radio control.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}

**Attachment:**
[Screen reader announcing 'Required' to every option.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3724498/Screen.reader.announcing.Required.to.every.option.zip)
![image](https://user-images.githubusercontent.com/55537167/66753000-694fc100-eeb0-11e9-894c-f7415e277001.png)

**Comment by khmakoto** [comment]
Hi @999-Dev, thanks for filing this issue! This is actually by design since we have specified that the `ChoiceGroup` is required, which leads to every radio button in the group to read the required prop. I hope this makes sense, but if it doesn't please let me know! In the meantime I'll be resolving this as `By Design`. Thanks again!
